torch/distributed/fsdp/flat_param.py

```diff
@@ -526,3 +526,14 @@ def _get_modules(self) -> Set[nn.Module]:
         return set(pi.module for pi in self.flat_param._param_infos).union(
             set(spi.module for spi in self.flat_param._shared_param_infos)
         )
+
+    def parameter_module_names(self) -> Iterator[Tuple[str, str]]:
```

Comment:
nit: docstring mentioning what is returned here?

Reply:
Miss this comment, will have another PR to fix the docstring.